linux_proc_extras/tests/test_integration.py

```diff
@@ -7,12 +7,20 @@
 
 from . import common
 
-pytestmark = pytest.mark.integration
-
 
+@pytest.mark.integration
 @pytest.mark.usefixtures("dd_environment")
 def test_check(aggregator, check):
     check.check(deepcopy(common.INSTANCE))
 
     for metric in common.EXPECTED_METRICS:
         aggregator.assert_metric_has_tag(metric, common.EXPECTED_TAG)
+
+
+@pytest.mark.e2e
+def test_check_e2e(dd_agent_check):
+    aggregator = dd_agent_check(deepcopy(common.INSTANCE), rate=True)
+    expected_metrics = deepcopy(common.EXPECTED_METRICS)
+    expected_metrics.remove('system.processes.priorities')
```

Comment:
Why is that metric removed/missing for e2e ? 🤔  `system.processes.priorities`

Reply:
In case this exclusion is required, do you mind adding an explanation comment ? thx

Reply:
The metric doesn't appear in the test environment we have for e2e. However, on a VM, the `system.processes.priorities` metric does work.